chsdi/views/transports.py

```diff
@@ -0,0 +1,72 @@
+# -*- coding: utf-8 -*-
+
+from sqlalchemy import and_
+from pyramid.view import view_config, view_defaults
+from chsdi.models.vector.uvek import Oev_departures
+import datetime
+
+
+@view_defaults(renderer='jsonp', route_name='transports')
+class TransportView(object):
+
+    def __init__(self, request):
+        self.request = request
+        if request.matched_route.name == 'transports':
+            self.id = request.matchdict['id']
+
+    @view_config(request_method='GET')
+    def get_departures(self):
+        current_date = datetime.datetime.now()
```

Comment:
You have to take into account time zones. Here is a way how to do it:
http://www.saltycrane.com/blog/2009/05/converting-time-zones-datetime-objects-python/

Otherwise we're 2 hours behind..
>>> strftime("%a, %d %b %Y %H:%M:%S +0000", gmtime())
'Tue, 20 Oct 2015 07:45:55 +0000'